### ISSUE TITLE
Google ReCaptcha is not appearing on registration screen

### ISSUE THREAD

**Issue by furqanbb** [buddyboss/buddyboss-platform#745]
**Describe the bug**
Using the Google reCaptcha - https://www.buddyboss.com/integrations/google-captcha-pro-recaptcha/ and enabled it for the Registration Form as shown in the screenshot.
https://prnt.sc/ryup6l
It doesn't appear on the sign-up page. http://prntscr.com/ryt4zb
But it appears on the login page.

**To Reproduce**
1. Install https://wordpress.org/plugins/google-captcha/ plugin and configure.
2. Enable the captcha for the registration page
3. View the registration page and see there is no captcha.

**Expected behavior**
The captcha should appear on the registration page too.

**Support ticket links**
https://secure.helpscout.net/conversation/1133466481/68013?folderId=3701248

**Comment by AiratHalitov** [comment]
Try to use this plugin: https://wordpress.org/plugins/advanced-nocaptcha-recaptcha/
It looks better for me.

**Comment by eisenwasser** [comment]
@AiratHalitov 
Thanks for the suggestion. 

We still want to support plugins that customers are using, the one in this issue was requested by a user in customer support.

That said, @furqanbb You can suggest to the customer to try:
https://wordpress.org/plugins/advanced-nocaptcha-recaptcha/

**Comment by AiratHalitov** [comment]
Style fix for `/wp-login.php` and `/registration/` pages for [the plugin]( https://wordpress.org/plugins/advanced-nocaptcha-recaptcha/)
```
#loginform .anr_captcha_field { margin-top: 40px; }
#signup-form .anr_captcha_field { margin-top: 20px; }
```

**Comment by manishpatolia** [comment]
@furqanbb 
https://wordpress.org/plugins/google-captcha/
This pro plugin Compatible with the BuddyPress registration form
https://prnt.sc/wgh0xw

So for captcha need to the pro version.

**Comment by emmanoctaviano** [comment]
@manishpatolia this has issues even on the Pro Version.

**Comment by manishpatolia** [comment]
@emmanoctaviano This issue is too old and when this issue created at that time this plugin has not a pro version, so the pro plugin issue and this both are different.

Pro plugin compatibility issue that already created in below:
https://github.com/buddyboss/buddyboss-platform/issues/745

@furqanbb So I'm closing this issue.

**Comment by bretweinraub** [comment]
I can confirm "Advanced noCaptcha & invisible captcha" [note the name change] is working reasonably well for BB Platform 1.8.2....

Note we are using WordFence Login Security on the login page...... so while the plugin recommends "all pages" for the google recaptcha scripts

![image](https://user-images.githubusercontent.com/214091/144338541-2dd4a103-90cf-439a-a5c9-700a404518e6.png)

for us, we are using form only as otherwise we end up with duplicate <script> tags.....

**Comment by MadCowWeb** [comment]
hoping i'm not speaking out of turn here, but this 100% does not work on the buddyBOSS platform. Perhaps it works swimmingly in buddyPRESS??  tried the pro version of CAPTCHA 4WP (https://www.wpwhitesecurity.com/wordpress-plugins/captcha-plugin-wordpress/pricing/) and nothing shows up on the buddyBOSS registration form. I added a ticket into buddyBOSS just to see if anyone over there might be working on a solution.
Just for kicks, we also tried the pro version of reCaptcha by BestWebSoft (https://bestwebsoft.com/products/wordpress/plugins/google-captcha/) and does not work.
